justfile

```diff
@@ -244,6 +244,10 @@ a:
 c:
     just catalog/up
 
+# alias for `just documentation/live`, 's' for Sphinx
+s:
+    just documentation/live
+
```

Comment:
I had no idea these aliases existed! I wish `d` was available :cry:

Reply:
I forget about them constantly too. 

Also, I used `d` first, and then `just` complained that it's already taken. `s` as in Sphinx was the next best thing.